packages/cache/src/internal/tar.ts

```diff
@@ -101,6 +101,7 @@ export async function createTar(
     }
   }
   const args = [
+    '--posix',
```

Comment:
I looked over the previous PR, I _think_ this won't cause any compat issues. No objections from me 👍

Reply:
Yeah, this shouldn't cause any problems.